src/isBetween.js

```diff
@@ -0,0 +1,22 @@
+var _curry3 = require('./internal/_curry3');
+
+
+/**
+ * Checks if a given number is between min and max values.
+ *
+ * @func
+ * @memberOf R
+ * @category Relation
+ * @sig Number -> Number -> Number -> Boolean
+ * @param {Number} min The min value.
+ * @param {Number} min The max value.
+ * @param {Number} min The value to be checked.
```

Comment:
Typo in name.  Should be `num` -- although IMHO that's not a great name.  Perhaps `test`